operational/vmware/instance_tag_sync/README.md

```diff
@@ -11,7 +11,7 @@ This policy has the following requirements to function properly. You will need t
 - RCAV
 - The cloud configured inside of CMP
 - [WS-Tunnel](https://github.com/rightscale/wstunnel)
-- `/usr/local/bin/wstunnel cli -token $WSTUNNEL_TOKEN -tunnel wss://wstunnel10-1.rightscale.com -server $VSPHERE_HTTPS -logfile /var/log/wstuncli-policy.log -pidfile /var/run/wstunnel-policy.pid`
+- `/usr/local/bin/wstunnel cli -token $WSTUNNEL_TOKEN -tunnel wss://wstunnel10-1.rightscale.com -server $VSPHERE_HTTPS -logfile /var/log/wstuncli-policy.log -pidfile /var/run/wstunnel-policy.pid --insecure`
```

Comment:
The `-insecure` flag should really only be used for local testing. Why was this added? As far as I can see `wstunnel10-1.rightscale.com` does have a TLS certificate that is currently valid.

Reply:
Yes @douglaswth , while testing the policy in Nimbus test we were required to use the insecure flag as it was giving error in accessing the vsphere. I thought user might be using the same vsphere so we can add the insecure flag. I have removed and merged the readme changes under this- https://github.com/flexera/policy_templates/pull/627, Thanks